roles/mongodb/tasks/main.yaml

```diff
@@ -74,36 +75,45 @@
     path: "/etc/tuned/virtual-guest-no-thp"
     owner: root
     group: root
+    mode: "0755"
 
 - name: Ensure tuned does not re-enable THP
   ansible.builtin.template:
     src: tuned.conf.j2
     dest: "/etc/tuned/virtual-guest-no-thp/tuned.conf"
+    mode: "0644"
 
 - name: Enable tuned profile
-  ansible.builtin.command: tuned-adm profile virtual-guest-no-thp
+  ansible.builtin.command:
+    cmd: tuned-adm profile virtual-guest-no-thp
   vars:
     ansible_python_interpreter: "{{ python_venv }}/bin/python3"
   ignore_errors: true
+  # register: result
+  # changed_when: result.rc == 0
+  # failed_when: result.rc > 0
 
 # Tune Kernel parameters
 - name: Adjust keepalive
   ansible.posix.sysctl:
     name: net.ipv4.tcp_keepalive_time
     value: 300
-  ignore_errors: true
+  failed_when: false
+  changed_when: false
```

Comment:
Doing this seems a little brute force.  We won't know if the value isn't updated or fails.  I don't see any examples in the ansible.posix.sysctl docs.  But I'm wondering if we should do a little testing to analyze the return value from ansible.

Reply:
Same with the other ansible.posix.sysctl tasks below.

Reply:
I tested this and I think you can just remove the ignore_errors, failed_when and changed_when completely